libs/components/lookup/testing/src/lookup/lookup-show-more-picker-harness.ts

```diff
@@ -0,0 +1,164 @@
+import { ComponentHarness, HarnessPredicate } from '@angular/cdk/testing';
+import {
+  SkyInfiniteScrollHarness,
+  SkyRepeaterItemHarness,
+} from '@skyux/lists/testing';
+
+import { SkySearchHarness } from '../search/search-harness';
+
+import { SkyLookupShowMorePickerHarnessFilters } from './lookup-show-more-picker-harness-filters';
+import { SkyLookupShowMorePickerSearchResultHarness } from './lookup-show-more-picker-search-result-harness';
+import { SkyLookupShowMorePickerSearchResultHarnessFilters } from './lookup-show-more-picker-search-result-harness-filters';
+
+/**
+ * Harness for interacting with a lookup's "Show more" picker in tests.
+ */
+export class SkyLookupShowMorePickerHarness extends ComponentHarness {
```

Comment:
Does this work with both the `search` modal and the `searchAsync` modal? If not and we have to pick one, I'd pick the async version since that's the one we support going forward.

Reply:
It does, but I reworked the tests so that both the standard and async pickers run the same suite of tests so we can be 100% sure.